conf.py

```diff
@@ -109,14 +109,22 @@
 
 # -- Options for HTML output ----------------------------------------------
 
-# The theme to use for HTML and HTML Help pages.  See the documentation for
```

Comment:
These are comments from standard config file and I want the sections that were modified to stand out. The name of the variable is pretty self-explanative, isn't it?